gui/resources.xml

```diff
@@ -1234,19 +1272,11 @@ Vocabulary
       <!-- {{{ Layers panel -->
       <child>
         <object class="GtkToggleAction" id="ToggleLayersTool">
-          <property name="label" translatable="yes" context="Menu→Window (labels), Toolbar (labels), Accel Editor (labels)">Layers Panel</property>
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Toggle the Layers dockpanel (arrange layers, or assign modes).</property>
+          <property name="label" translatable="yes" context="Menu→Layers (lables), Menu→Window (labels), Toolbar (labels), Accel Editor (labels)">Layers Panel</property>
+          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Dockable panel for managing layers.</property>
           <property name="icon-name">mypaint-layers-symbolic</property>
           <signal name="activate" handler="toggle_dockpanel_cb"/>
         </object>
-      </child>
-      <child>
-        <object class="GtkAction" id="RevealLayersTool">
-          <property name="label" translatable="yes" context="Menu→Layers (labels), Accel Editor (labels)">Show Layers Panel</property>
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Reveal the Layers dockpanel (arrange layers, or assign modes).</property>
-          <property name="icon-name">mypaint-layers-symbolic</property>
-          <signal name="activate" handler="reveal_dockpanel_cb"/>
-        </object>
```

Comment:
They don't do the same thing. The toggle version shuffles the layout.
You may regard it as a trivial difference, but from my point of view, that single difference makes 'Toggle layer panel' an action I never want to invoke. 

Being a little more objective and focusing on use cases: I see no reason why either
a) a user who has currently fullscreened MyPaint would want to invoke Toggle rather than Reveal
b) a user who is in non-fullscreen mode would want to invoke Reveal rather than Toggle.

It would be nice to have an action that *just worked* for all cases. But I don't think there is any realistic argument to be made that we currently have such an action.